suricata.yaml.in

```diff
@@ -916,6 +917,12 @@ app-layer:
       stream-depth: 0
       # Maximum number of live PostgreSQL transactions per flow
       # max-tx: 1024
+    mysql:
+      enabled: no
+      # Stream reassembly size for MySQL. By default, track it completely.
+      stream-depth: 0
```

Comment:
Why that default ?

Reply:
I copied from pgsql configuration, now i know what it means, i will remove it.